Source/WebCore/animation/KeyframeEffect.cpp

```diff
@@ -2287,7 +2287,7 @@ std::optional<double> KeyframeEffect::progressUntilNextStep(double iterationProg
 
 bool KeyframeEffect::ticksContinouslyWhileActive() const
 {
-    auto doesNotAffectStyles = m_blendingKeyframes.isEmpty() || m_blendingKeyframes.properties().isEmpty();
+    auto doesNotAffectStyles = m_blendingKeyframes.isEmpty() || m_blendingKeyframes.properties().isEmpty() || !renderer();
```

Comment:
`display:contents` boxes also don't have a renderer. Should those animations still tick? (you could test `element.hasDisplayContents()`).